lib/image_optim/config.rb

```diff
@@ -157,6 +157,11 @@ def cache_worker_digests
       !!get!(:cache_worker_digests)
     end
 
+    def timeout
+      timeout = get!(:timeout)
+      timeout ? timeout.to_i : 0
```

Comment:
If any non-integer value is appropriate, it should be `nil` -- at least that has built-in support for testing equality.